senior-knights/course-schedulizer
# Allow export of old and new formats

To Test:
- `npm i && npm start`
- Try importing the full course schedule, the math schedule, etc.
- There should now be two export buttons, try them both:
  - `EXPORT CSV` should give two CSVs in the new formats described by VanderLinden in our most recent meeting (it doesn't have the extra headers, but I figure the CSV should be good enough and allow department chairs to just copy the contents over)
  - `EXPORT FULL CSV` should do what `EXPORT CSV` used to do

Worth noting that the non-teaching CSV given is composed of classes without Prefix and Number, unsure if this is what we want to do going forward. (Currently it is impossible to add such a "course", so eventually we will want an add non-teaching load option, but it might still be good to store this as a course/section without prefix and number).

Let me know if the wording is confusing, if we should just remove full export or not, and if we should wait to merge this until we know more about the new formats.

Additionally, this branch changes the test CSV files to be loaded from their current version rather than current `develop` (I should have done it that way in the first place)

Closes #80
